admin/appearance.md

```diff
@@ -6,20 +6,34 @@ description: Learn how to augment the dashboard appearance.
 Coder offers you appearance customization options for the following:
 
 - System banner messages
+- Service banner messages
 - The terms of service display
 
-## System banner messages
+## System banners
 
 To customize your system banner messages:
 
 1. Go to **Manage** > **Admin** > **Appearance** in the Coder UI.
-1. Toggle the switch to **On**.
+1. Toggle the **System Banner** switch to **On**.
 1. Set your **Background Color**, and provide the text you want to be displayed
    in your **Header** and **Footer**.
 1. Click **Save Preferences** to save your changes.
 
 ![System appearance](../assets/system-banners.png)
 
+## Service banners
+
+The service banner allows you to display a message to all users of your Coder
+system. The user can dismiss the message at any time and Coder will not display
+a banner until you change the message.
```

Comment:
To be explicit, the logic is:

- if the same message is disabled, then re-enabled, all users will see it
- if the message is updated, all users will see it

I don't know if we should clarify that in the wording, or this is satisfactory - doing a gut check, entirely up to you.